### PR TITLE
Focused-Launch: fix styling for disabled Free plan card

### LINKED ISSUE #47788: Focused Launch: Plans Grid unavailable Free plan copy & styles in Plan details view
Currently used in Focused Launch - Plans Details view, the version of `<PlansTable/>` should have specific styling & copy for unavailable plans but Production doesn't match the specs specified in Figma. See the screenshots below for comparison.

#### Acceptance Criteria

* [ ] The cross icon's color for the unavailable plan is orange (`--studio-orange-40`).
* [ ] The unavailable domain copy is crossed through.
* [ ] The disabled button's copy is: `Unavailable with domain`
* [ ] The orange copy is: `[domain] is not included`

#### Additional information
Button styles are tackled separately in #47740.

#### What I expected

Screenshot 2020-11-27 at 13 27 52

#### Current situation on Production
Screenshot 2020-11-26 at 11 24 16

### PR DESCRIPTION
**Pull request by StefanNieuwenhuis**
#### Changes proposed in this Pull Request

The current styling & copy for unavailable plans (e.g. free plan with a paid domain) in `<PlansTable/>` are unaligned with Design Specs, described in Figma. In this PR, styling and copy changes are proposed to align them. 

#### Technical changes
* Update fill & stroke color for cross-icon.
* Cross through unavailable domain copy.
* Update unavailable domain's font-weight.
* Update copy for disabled plan's button.
* Update copy for the unavailable domain (text after cross-icon).

#### Focused Launch
1. Go to `/page/UNLAUNCHED_SITE_ID_CREATED_WITH_START.wordpress.com/home?flags=create/focused-launch-flow-calypso`.
2. Click "Launch" to open the Focused Launch flow.
3. Select a **paid domain**
4. Click "View all plans"
5. Check that the Free plan's:
    * [ ] Button is disabled
    * [ ] The disabled button's copy is: `Unavailable with domain`
    * [ ] Cross-icon is `--studio-orange-40`
    * [ ] The orange copy is: `[domain] is not included`

#### Gutenboarding - Regression testing
1. Go to `/new`.
2. Enter a site name, pick a paid domain (preferably a `.blog` domain)
3. Proceed to the step "Select a plan".
4. Expand the free plan and check that:
    * [ ] Cross-icon is `--studio-orange-40`
    * [ ] The orange copy is: `[domain] is not included`

#### Editing Toolkit Step-by-Step Launch - Regression testing
1. Check out this branch and run `yarn dev --sync` from `/apps/editing-toolkit` ensuring you're also logged into your sandbox environment.
2. Go to `/new` and go through the process to create a new site, you will land in the editing experience.
3. Sandbox your newly created site in your host file.
4. Flush your cache & DNS and then refresh your site made in step 1 and ensure that you're using the `dev` version of the ETK.
5. Click "Launch". This should open the launch modal.
6. Select a **paid domain** at the "Select a domain" step
7. Proceed to the step "Select a plan" and confirm that:
    * [ ] No unexpected changes have appeared.

Fixes #47788

### CODE REVIEW

**Review by ciampo — DISMISSED** [review]
Please update this PR with the updated requirements listed in #47788

**Comment by ciampo** [comment]
I updated the requirements as per https://github.com/Automattic/wp-calypso/issues/47814#issuecomment-734650649

**Comment by StefanNieuwenhuis** [comment]
> I updated the requirements as per #47814 (comment)

Thanks @ciampo! I'll update it accordingly.

File: packages/plans-grid/src/plans-feature-list/style.scss
Comment (ciampo) [inline packages/plans-grid/src/plans-feature-list/style.scss]: Bit of a nitpick, but it would interesting to see if we can use a combination of CSS inheritance and the `currentColor` keyword to avoid specifying the same variables more than once:


```scss
&--disabled-message {
  color: var( --studio-orange-40 );

  & > svg path {
    fill: currentColor;
    stroke: currentColor);
  }

  & > span {
    font-weight: bold;
    color: inherit
    text-decoration: line-through;
  }
}
```

What do you think?

File: packages/plans-grid/src/plans-feature-list/style.scss
Comment (ciampo) [inline packages/plans-grid/src/plans-feature-list/style.scss]: Why are we changing the color of the tick in this case from green to gray? From the looks of it, we should only change the color of the tick from gray to orange, when the free plan is not available

I'm just trying to be mindful that this component is used across different flows, and that we need to make sure that we don't break those flows by making changes :)

File: packages/plans-grid/src/plans-table/style.scss
Comment (ciampo) [inline packages/plans-grid/src/plans-table/style.scss:28]: Probably related to [this other comment of mine](https://github.com/Automattic/wp-calypso/pull/47796#discussion_r531588492), but why are we deleting this line?

**Review by ciampo — COMMENTED** [review]
Also just left a few comments, knowing that you're already working on a new version of this PR

File: packages/plans-grid/src/plans-feature-list/style.scss
Comment (StefanNieuwenhuis) [reply packages/plans-grid/src/plans-feature-list/style.scss]: According to design specs, all checks have to be dark gray hence I updated the colors. Can you verify if this is the actual case or if I'm mistaking? :)

File: packages/plans-grid/src/plans-table/style.scss
Comment (StefanNieuwenhuis) [reply packages/plans-grid/src/plans-table/style.scss:28]: It's indeed related to [your other comment](https://github.com/Automattic/wp-calypso/pull/47796#discussion_r531588492). When all ticks have to be dark gray, there's no need for this selector anymore.

**Comment by ciampo** [comment]
> Please update this PR with the updated requirements listed in #47788

Hey @StefanNieuwenhuis , it looks like this PR still satisfies the "old" requirements. Can you please update this PR to include the updated copy? Thank you!

**Comment by StefanNieuwenhuis** [comment]
> Hey @StefanNieuwenhuis , it looks like this PR still satisfies the "old" requirements. 

My bad! And here's a follow-up question: It makes sense to me to (re)use `disabledLabel` for the copy but, as you can see on the screenshot, the copy doesn't meet the specs.

Screenshot 2020-11-30 at 10 12 26

Three options:
1. Update the copy in `disabledLabel` but that might have unwanted side-effects. 
2. Leave the copy as is and deviate from specs. 
3. Create a new copy and align it with specs.

@ciampo What do you think?

File: packages/plans-grid/src/plans-feature-list/style.scss
Comment (ciampo) [reply packages/plans-grid/src/plans-feature-list/style.scss]: It looks like this change does have an impact.

**This is how the plans grid looks like on production currently:**

In New Onboarding flow:

Screenshot 2020-11-30 at 11 35 21

In Step by Step flow:

Screenshot 2020-11-30 at 11 39 42


**And this is how it looks like on this PR**

In New Onboarding flow:

Screenshot 2020-11-30 at 11 36 36

In Step by Step flow:

Screenshot 2020-11-30 at 11 42 38

---

As you can see, in production all ticks are green. Is there a way we can avoid this breaking change, since we're only supposed to change the color of the disabled plan icon to orange?

**Comment by ciampo** [comment]
>@ciampo What do you think?

As we agreed in a previous call, the next steps are:

1. [x] Rebase OR merge trunk into PR
2. [x] Revert tick color changes
3. [x] Investigate where the `disabledLabel` variable is used, and discuss with Ollie the best approach (ideally we'd like to replace the copy without creating new exceptions)
4. [x] Make sure that all required criteria are met
5. [x] Re-add the "Needs Review" label and ask for the next round of reviews

**Comment by StefanNieuwenhuis** [comment]
@ollierozdarz, the suggested disabled button's copy is different from the plan accordion's disabled plan copy (see screenshots).
What do you recommend? Do we want to stay consistent and use the same copy everywhere or do we want to introduce an exception (which makes the code more complex and less maintainable as well)? 


Screenshot 2020-11-26 at 12 58 31

PlansAccordionView

**Review by ciampo — COMMENTED** [review]
Changes LGTM, but before merging, we may want to wait for final feedback about how these changes affect other flows (gutenboarding, step by step launch)

**Comment by ciampo** [comment]
Hey @ollierozdarz, before merging we'd like you to take a final look at how these changes are affecting flows using the plans grid (see [this comment](https://github.com/Automattic/wp-calypso/issues/47788#issuecomment-759323920) for more details)

Thank you!

**Comment by ollierozdarz** [comment]
> Hey @ollierozdarz, before merging we'd like you to take a final look at how these changes are affecting flows using the plans grid (see [this comment](https://github.com/Automattic/wp-calypso/issues/47788#issuecomment-759323920) for more details)

@ciampo these language changes look good to me!

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/5462203

Hi @StefanNieuwenhuis, could you please edit the description of this PR and add a screenshot for our translators? Ideally it'd include this string: `Unavailable with domain`

Thank you in advance!

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/5462203).

**Pull request merged**
